packages/core/src/utils/cookie.ts

```diff
@@ -10,7 +10,7 @@ export const cookieStorage = {
   },
   setItem(key, value) {
     if (typeof window === 'undefined') return
-    document.cookie = `${key}=${value}`
+    document.cookie = `${key}=${value};path=`
```

Comment:
what's the advantage of `path=` over `path=/`?

Reply:
It's a mistake... fixed.